EventKit/eventkit-cloud
# Bump python django

## Description of Improvement

Updates Python Dependencies

## How to test

Build the conda deps, and rebuild the docker images.  Run the tests, and try the application to ensure it works as expected. 

## Quality Assurance 

The following items have been updated:
 
## Screenshots



:smile:
